tests/utils/inputparams/testparams.go

```diff
@@ -42,10 +43,14 @@ func init() {
 func GetVolumeName() string {
 	return volumeName
 }
-
 // GetVolumeNameWithTimeStamp prepares unique volume name by appending current time-stamp value
-func GetVolumeNameWithTimeStamp(volumeName string) string {
-	return volumeName + "_volume_" + strconv.FormatInt(time.Now().Unix(), 10)
+func GetVolumeNameWithTimeStamp(volName string) string {
+	return volName + "_volume_" + strconv.FormatInt(time.Now().Unix(), 10)
+}
+
+// GetVolumeName prepares unique volume name with a random generated number
+func GetNewVolumeName(volName string) string {
```

Comment:
why can not we stick with `GetVolumeNameWithTimeStamp()`?  is there any benefit for introducing this new method?